cpp/test/neighbors/ann_cagra.cuh

```diff
@@ -892,11 +893,30 @@ inline std::vector<AnnCagraInputs> generate_inputs()
     {true},
     {0.995});
 
+  // Fixed dim, and changing neighbors and query size (output matrix size)
   auto inputs2 = raft::util::itertools::product<AnnCagraInputs>(
+    {1, 100},
```

Comment:
Above we tested with different `max_query` parameter, here we test with batch size 1 and 100.